### ISSUE TITLE
I suggest renaming the repository to Invasion-TotalDefense

### ISSUE THREAD

**Issue by presmihaylov** [gkkirilov/SoftUni-OOP-Teamwork#2]
That way, it will be a lot more marketable and good looking.

**Comment by georgiai1** [comment]
Thanks for the suggestion @PreslavMihaylov.
What about Invasion Tower Defense so it suggests the game genre?

**Comment by presmihaylov** [comment]
Yeah, that's a good idea. However, in that case you will have to change the folder name to the same name as well.

**Comment by presmihaylov** [comment]
And perhaps the game title in-game
